### PR TITLE
[update] update website/doc , remove '$' in shell

### PR DESCRIPTION
**Pull request by fewdan**
### What problem does this PR solve?

update website/doc , remove '$' in shell

### CODE REVIEW

File: pkg/apiserver/event/event.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 // @Param experimentNamespace query string false "The namespace of the experiment"
 // @Param uid query string false "The UID of the experiment"
 // @Param kind query string false "kind" Enums(PodChaos, IoChaos, NetworkChaos, TimeChaos, KernelChaos, StressChaos)
+// @Param limit query string false "The max length of events list"
```
Comment (yeya24) [inline pkg/apiserver/event/event.go:74]: Are these limit changes related?

Comment (fewdan) [reply pkg/apiserver/event/event.go:74]: @yeya24   please see https://github.com/chaos-mesh/chaos-mesh/pull/742  .
I made a mistake when switching branches.
